.gitignore

```diff
@@ -49,3 +49,4 @@ package-lock.json
 npm-debug.log
 .vscode
 .yo-rc.json
+.vendor/*
```

Comment:
which file is that? I'm wondering if its too confusing as we do have a vendor folder.